### PR TITLE
Lecture 08 (dansl / Daniel Liu)

### PR DESCRIPTION
**Pull request by btsp6**
I tried my best but the latter part of the lecture is a little too hardware technical for me to grok :( I will accept any point penalties.

### CODE REVIEW

File: notes/lecture-08-dansl.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+Moreover, we can choose a different optimal subnetwork based on the target hardware, based on the computational capabilites of that hardware. This way, we don't need to train a new model every time we want to create a model for a specific piece of hardware. 
+
+One concern is that the subnetworks may not perform very well. Fine tuning on the chosen subnetwork can be done to alleviate this. 
+
+Surprisingly, we can get many sub-networks from a single Once for All network, over $10^{19}$.
+
+## Progressive Pruning
+
+If we want to create a sub-network with a smaller kernel size, depth, or channel width, we can use progressive pruning. We can reduce
```
Comment (zhijian-liu) [inline notes/lecture-08-dansl.md:77]: This sentence is not complete.

**Review by zhijian-liu — CHANGES_REQUESTED** [review]
Thanks for your contribution! The scribe looks great. Please
* Add the note information section as in https://github.com/mit-han-lab/6s965-fall2022/blob/main/notes/lecture-01-johndoe.md
* Address the comments below.

Please re-request the review when you find this PR ready. Btw, feel free to come to the office hour. We would be happy to explain more about the hardware part.

**Comment by zhijian-liu** [comment]
@btsp6, please let us know when the scribe is ready for another round of review.

**Comment by btsp6** [comment]
Sorry for the delay, I've updated the requested changes.

**Review by zhijian-liu — APPROVED** [review]

**Pull request merged**